pkg/lightning/backend/local/local.go

```diff
@@ -854,19 +855,19 @@ func (local *local) WriteToTiKV(
 		zap.Int64("buf_size", bytesBuf.totalSize()),
 		zap.Stringer("takeTime", time.Since(begin)))
 
-	var remainRange *Range
+	finishedRange := regionRange
 	if iter.Valid() && iter.Next() {
 		firstKey := append([]byte{}, iter.Key()...)
-		remainRange = &Range{start: firstKey, end: regionRange.end}
+		finishedRange = Range{start: regionRange.start, end: firstKey}
```

Comment:
Shall it be `regionRange` or `start` passed by method parameter?

Reply:
The `start`, `end` passed from method parameter is the Range calculated by lightning, and it's range may covers multiple regions. The`regionRange` is the actual range the intersect  target region and [start, end). So the finishedRange's start should be the regionRange's start